### ISSUE TITLE
Level: "Dig For Victory"

### ISSUE THREAD

**Issue by tttppp** [andybalaam/rabbit-escape#47]
Not an outside level, but I liked how this looks.
```
:name=Dig For Victory
:description=Can you navigate the warren?
:num_rabbits=10
:num_to_save=10
:dig=7
:music=tryad-witness
#########
# Q # Q #
#(((#)()#
#()(#(((#
#)()((((#
#)))())(#
#(()((()#
#)()()()#
#(()))()#
#)))()()#
#)())))(#
#()) ())#
#### ####
###   ###
### # ###
### O ###
#########
```

**Comment by colonelfazackerley** [comment]
It looks great when you first open it. The rabbits don't move around it very nicely, though. Perhaps open it up a bit? Maybe don't have a bridge section in _every_ square.

I completed it first time, but I am not sure I would do it every time. I don't recall my solution.

You are on fire with the originality.

**Comment by tttppp** [comment]
Yep, I agree that it's not obvious where the rabbits will go. I've won a few times and lost a few, and wonder if it could be solved with fewer diggers.

I also found a perpetual digging bug while creating this level where a rabbit that was given one or two tokens would dig through all the bridges and off the bottom of the world. I also one time had a levitating rotating rabbit at the top and bottom of the screen. I can't reproduce these issues though.

**Comment by colonelfazackerley** [comment]
I had one spare dig token at the end.

**Comment by tttppp** [comment]
Just solved it using six dig tokens - I think that's really quite hard.  I'm not sure about how to make the level have more space, and still work well. I just had a go but the level didn't look very good/work very well.

**Comment by colonelfazackerley** [comment]
I like it for it's originality. Don't know if animation tweaks, like those needed for the saw level might help.

Let's see what Andy thinks.

**Comment by andybalaam** [comment]
I haven't had a chance to play it yet, but I am inclined to include it in a later level set, since it's not very outdoors-y.  If we don't have enough outdoors levels, I might change my mind.

**Comment by colonelfazackerley** [comment]
I think we should aim for 2 sets of 20 new levels for the next feature release.

**Comment by andybalaam** [comment]
Woah!  If we manage that many levels, I will definitely split that across 2 releases to make it feel like an active project (which it is!).  By the way, I am working on hints, which will be the only significant feature in the next release.  The one after that will have levels.

**Comment by colonelfazackerley** [comment]
close me.
merged to master with f96dd5696bc0d3c0f44b4856ba75e3a7d2bc201a

**Comment by andybalaam** [comment]
Congratulations on designing the most annoying level so far! I really like it and am ok with the animation bugs it exposes. I solved it with 1 token left after many failed attempts. I imagine when we introduce minor behaviour changes any solutions we have for this will break,  but that's its charm.
